docs/components/best-practices/architecture/sizing-your-environment.md

```diff
@@ -143,19 +143,31 @@ Now you can select a hardware package that can cover these requirements. In this
 
 ### Camunda 8 SaaS
 
-Camunda 8 defines three fixed hardware packages you can select from. The table below gives you an indication what requirements you can fulfill with these. If your requirements are above the mentioned numbers, please contact us to discuss a customized sizing.
+Camunda 8 defines four fixed hardware packages you can select from (1x, 2x, 3x, 4x) when choosing your cluster [type](/components/concepts/clusters.md#cluster-type) and [size](/components/concepts/clusters.md#cluster-size). The following table gives you an indication of what requirements you can fulfill with each cluster size.
```

Comment:
Before merging: We should check if we still have time to benchmark the 4th. As in the beginning we aligned on 3, the benchmark so far was only made with 3 multiplication. 

If after having setup 3 the zeebe team agrees to have time to benchmark a multiplication of 4 + console team added the 4th bottom, we can go with that (it should not be too much effort - but its effort ;) ).

cc @felix-mueller